packages/ketcher-react/src/script/editor/utils/customOnChangeHandler.ts

```diff
@@ -40,7 +40,9 @@ type Data = {
 
 export function customOnChangeHandler(action, handler) {
```

Comment:
For a fail safe check I'd add type for `action` parameter – let it be `Action | undefined` for now, as we could see, it's possible to trigger event without any data and it will silently fail on the following `forEach` call